gulpfile.js

```diff
@@ -11,84 +11,92 @@ plugins.del            = require("del");
-	return plugins.del(['dist/css/patternlab/*'],cb);
+	return plugins.del(['dist/styleguide/css/*'],cb);
 });
 
 gulp.task('clean:css-custom', function (cb) {
-	return plugins.del(['dist/css/custom/*'],cb);
+	return plugins.del(['dist/styleguide/css/custom/*'],cb);
 });
 
 gulp.task('clean:fonts', function (cb) {
-	return plugins.del(['dist/fonts/*'],cb);
+	return plugins.del(['dist/styleguide/fonts/*'],cb);
 });
 
 gulp.task('clean:html', function (cb) {
-	return plugins.del(['dist/html/*'],cb);
+	return plugins.del(['dist/*'],cb);
 });
 
 gulp.task('clean:images', function (cb) {
-	return plugins.del(['dist/images/*'],cb);
+	return plugins.del(['dist/styleguide/images/*'],cb);
 });
 
 gulp.task('clean:js', function (cb) {
-	return plugins.del(['dist/js/*'],cb);
+	return plugins.del(['dist/styleguide/js/*'],cb);
 });
 
 /* core tasks */
 gulp.task('build:bower', ['clean:bower'], function(){
 	return gulp.src(plugins.mainBowerFiles())
 		.pipe(plugins.rename({suffix: '.min'}))
 		.pipe(plugins.uglify())
-		.pipe(gulp.dest("dist/bower_components"))
-		.pipe(gulp.dest("../../../public/styleguide/bower_components"));
+		.pipe(gulp.dest("dist/styleguide/bower_components"))
+		.pipe(copyPublic("styleguide/bower_components"));
 });
 
 gulp.task('build:css-general', function() {
 	return gulp.src(['src/css/prism-okaidia.css','src/css/typeahead.css'])
 		.pipe(plugins.concat('prism-typeahead.css'))
-		.pipe(gulp.dest('dist/css/patternlab'))
+		.pipe(gulp.dest('dist/styleguide/css'))
 		.pipe(plugins.rename({suffix: '.min'}))
 		.pipe(plugins.minifyCss())
-		.pipe(gulp.dest('dist/css/patternlab'))
-		.pipe(gulp.dest('../../../public/styleguide/css'));
+		.pipe(gulp.dest('dist/styleguide/css'))
+		.pipe(copyPublic("styleguide/css"));
 });
 
 gulp.task('build:css-patternlab', ['clean:css-patternlab', 'build:css-general'], function() {
 	return plugins.rubySass('src/sass/styleguide.scss', { style: 'expanded', "sourcemap=none": true })
 		.pipe(plugins.autoprefixer({browsers: ['last 2 version', 'safari 5', 'ie 8', 'ie 9', 'opera 12.1', 'android 4']}))
-		.pipe(gulp.dest('dist/css/patternlab'))
+		.pipe(gulp.dest('dist/styleguide/css'))
 		.pipe(plugins.rename({suffix: '.min'}))
 		.pipe(plugins.minifyCss())
-		.pipe(gulp.dest('dist/css/patternlab'))
-		.pipe(gulp.dest('../../../public/styleguide/css'));
+		.pipe(gulp.dest('dist/styleguide/css'))
+		.pipe(copyPublic("styleguide/css"));
 });
 
 gulp.task('build:css-custom', ['clean:css-custom'], function() {
 	return plugins.rubySass('src/sass/styleguide-specific.scss', { style: 'expanded', "sourcemap=none": true })
 		.pipe(plugins.autoprefixer({browsers: ['last 2 version', 'safari 5', 'ie 8', 'ie 9', 'opera 12.1', 'android 4']}))
-		.pipe(gulp.dest('dist/css/custom'))
+		.pipe(gulp.dest('dist/styleguide/css/custom'))
 		.pipe(plugins.rename({suffix: '.min'}))
 		.pipe(plugins.minifyCss())
-		.pipe(gulp.dest('dist/css/custom'))
-		.pipe(gulp.dest('../../../source/styleguide/css'));
```

Comment:
@dmolsen This also seems to be a "one-off" or special case that we might need to address?